<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMS</class>
 <widget class="QMainWindow" name="TMS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
font: 10pt &quot;푸라닭 젠틀 고딕&quot;;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="localBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>1160파주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1120청주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1110중앙</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1130구미</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1140광주</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1142여수</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1150창원</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1141익산</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1190미국</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1151울산</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;푸라닭 젠틀 고딕&quot;;
</string>
    </property>
    <property name="text">
     <string>TMS - ACO</string>
    </property>
   </widget>
   <widget class="QLabel" name="local">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;푸라닭 젠틀 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>지역</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="no_over_cap">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>과적 금지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>590</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>실행</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="no_over_time">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>초과 근무 금지</string>
    </property>
   </widget>
   <widget class="QLabel" name="quantity">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>284</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>물량 비용</string>
    </property>
   </widget>
   <widget class="QLabel" name="hub_radius">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>314</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>반경 비용</string>
    </property>
   </widget>
   <widget class="QLabel" name="crossline">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>344</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>교차 비용</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="max_capacity_cost">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="max_capacity">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>최대 적재</string>
    </property>
   </widget>
   <widget class="QLabel" name="limit_capacity">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>제한 용량</string>
    </property>
   </widget>
   <widget class="QLabel" name="limit_time">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>최대 시간</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="limit_capacity_var">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="limit_time_var">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="max_capacity_cost_var">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>최대 중량 비용</string>
    </property>
   </widget>
   <widget class="QLabel" name="timewindow">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>시간 초과 비용</string>
    </property>
   </widget>
   <widget class="QLabel" name="car_speed">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>차량 속도(km/h)</string>
    </property>
   </widget>
   <widget class="QLabel" name="loading_unloading">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>상하차 시간(m)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="loading_unloading_var">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="used_limit_time">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사용된 근무 시간의</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="used_limit_time_var">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="value">
     <double>0.800000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="used_limit_time2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>% 를 충족하면 차량을 제외</string>
    </property>
   </widget>
   <widget class="QLabel" name="used_limit_cap2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>% 를 충족하면 차량을 제외</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="used_limit_cap_var">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>340</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <double>0.800000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="used_limit_cap">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사용된 차 용량의</string>
    </property>
   </widget>
   <widget class="QLabel" name="car_customer">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>차와 지점간의 거리 비용</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="quantity_var">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QSpinBox" name="hub_radius_var">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="crossline_var">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="car_customer_var">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="max_capacity_cost_var_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="timewindow_var">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="car_speed_var">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="available_yong">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>용차 사용</string>
    </property>
   </widget>
   <widget class="QLabel" name="count">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>반복 횟수</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="count_var">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>390</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="initial_pheromone">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>초기 페로몬</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="initial_pheromone_var">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>43</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="deposit_pheromone">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>470</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>페로몬 축적 정도</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pheromone_deposit_weight">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>43</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="evaporation_pheromone_var">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>43</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="eva_phe">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>510</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>페로몬 증발 정도</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
